docs/data-guidelines.md

```diff
@@ -280,3 +280,23 @@ Members of this mixin are available to `HTMLAnchorElement` and `HTMLAreaElement`
    ```
 
 This guideline was proposed in [#8929](https://github.com/mdn/browser-compat-data/issues/8929), based in part on previous discussion in [#472](https://github.com/mdn/browser-compat-data/issues/472).
+
+## Choosing an experimental status
+
+Generally, when a feature is supported by one and only one browser engine, set `experimental` to `true`. When a feature is supported by two or more engines, then set `experimental` to `false`. Some exceptions apply, however, for long-standing features and features behind flags and prefixes.
+
+If a feature is supported behind flags only, no matter how many engines, then set `experimental` to `true`.
```

Comment:
Note that I _carefully_ avoided the word "implementation" throughout this guideline. We have proposals to handle stuff in development with things `implementation_url`. It seems to me that if it's not on by default, it doesn't count.

Reply:
Completely agree where Chrome is concerned. This is especially true in light of more recent use of our flag mechanism, which is now [explicitly earlier than an origin trial](https://www.chromium.org/blink/launching-features), which are not official parts of the web platform.